packages/web-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,199 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorBrandBackground,
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorNeutralBackground1,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeDisabled,
+  colorStrokeFocus2,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-grid')} :host {
+    --thumb-size: 18px;
+    --thumb-padding: 3px;
+    --thumb-translate: calc(var(--thumb-size) * -0.5 + var(--track-width) / 2);
+    --track-overhang: -2px;
+    --track-width: 4px;
+    --fast-slider-height: calc(var(--thumb-size) * 10);
+    --slider-direction: 90deg;
```

Comment:
These end up as global variables - is that desired on the Fluent side of things (does this exist in React as well or is this net-new delta for WC's)?

Reply:
Is there a way to scope them to only this component? They're pretty helpful for reducing the amount of CSS required and can come in handy when switching between how attributes affect the appearance.

Reply:
@chrisdholt Following up here. Will these be scoped to the component since they are in the :host selector? At least that was my understanding.